README.md

```diff
@@ -288,3 +288,14 @@ $ curl -X POST \
   }
 }'
 ```
+
+### <a name="using-comment"></a>Query Posts with Comments
+
+#### GraphQL
+
```

Comment:
Need GraphQL one too.

```gql
post {
  edges {
     node {
       id
       title
       comments {
         edges {
           node {
             id
             content
          }
        }
      }
    }
}
```
or something similar (I forget the exact model atm 😄 )

Reply:
Added